Library/Formula/scirius.rb

```diff
@@ -0,0 +1,126 @@
+    sha1 "ef3c2e9d7fb2961cee41399607589fcb9eb06b63"
+  end
+
+    resource "gitdb" do
+    url "https://pypi.python.org/packages/source/g/gitdb/gitdb-0.6.4.tar.gz"
+    sha1 "21cbba28199802e73e0c4a83b7c02369bbf8c7d7"
+  end
+
+    resource "GitPython" do
+    url "https://pypi.python.org/packages/source/G/GitPython/GitPython-1.0.0.tar.gz"
+    sha1 "0c102b969973dd055488e519e1854094533c5cfe"
+  end
+
+    resource "psutil" do
+    url "https://pypi.python.org/packages/source/p/psutil/psutil-2.2.1.tar.gz"
+    sha1 "ddf58b3a0e699e142586b67097e3ae062766f11d"
+  end
+
+      resource "Pygments" do
+    url "https://pypi.python.org/packages/source/P/Pygments/Pygments-2.0.2.tar.gz"
+    sha1 "fe2c8178a039b6820a7a86b2132a2626df99c7f8"
+  end
+
+    resource "pytz" do
+    url "https://pypi.python.org/packages/source/p/pytz/pytz-2015.2.tar.bz2"
+    sha1 "9a1b8bf9b1222f63f7d677f124453d13d7ed27c8"
+  end
+
+    resource "requests" do
+    url "https://pypi.python.org/packages/source/r/requests/requests-2.6.0.tar.gz"
+    sha1 "ad7327c73e8be8c188ad489d511097202b1fef12"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha1 "d168e6d01f0900875c6ecebc97da72d0fda31129"
+  end
+
+  resource "smmap" do
+    url "https://pypi.python.org/packages/source/s/smmap/smmap-0.9.0.tar.gz"
+    sha1 "2aad63033aab4e4cf1a3f75f521d985da8b1353f"
+  end
+
+    resource "South" do
+    url "https://pypi.python.org/packages/source/S/South/South-1.0.2.tar.gz"
+    sha1 "8f9bf809aa8c63e5b46359d3751b76786b9f58c6"
+  end
+
+    resource "urllib3" do
+    url "https://pypi.python.org/packages/source/u/urllib3/urllib3-1.10.1.tar.gz"
+    sha1 "434ff4f51b566076ed2f41ff71f24e1b0d5c9ebd"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[daemon Django django-bootstrap3 django-revproxy django-tables2 gitdb GitPython psutil Pygments pytz requests six smmap South urllib3].each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+#
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+#
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+plist_options :manual => "python"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+        <dict>
+            <key>KeepAlive</key>
+            <true/>
+            <key>Label</key>
+            <string>#{plist_name}</string>
+            <key>ProgramArguments</key>
+            <array>
+                <string>#{HOMEBREW_PREFIX}/bin/python</string>
```

Comment:
again, you mean to use system `/usr/bin/python` instead of `/usr/local/bin/python`? But if someone installs scirius on old os x (let's say leopard)))) or on machine without system python? In that case plist should look different because it uses different location for `python`. And the Launch Agent won't work with just `<string>python</string>` because it needs an absolute path... Correct me if I'm wrong please